docs/trace/exception-handling/Program.cs

```diff
@@ -0,0 +1,76 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+
+public class Program
+{
+    private static readonly ActivitySource MyActivitySource = new ActivitySource(
+        "MyCompany.MyProduct.MyLibrary");
+
+    public static void Main()
+    {
+        AppDomain.CurrentDomain.UnhandledException += UnhandledExceptionHandler;
+
+        using var tracerProvider = Sdk.CreateTracerProviderBuilder(options =>
+            {
+                options.SetErrorStatusOnException = true;
+            })
+            .AddSource("MyCompany.MyProduct.MyLibrary")
+            .SetSampler(new AlwaysOnSampler())
+            .AddConsoleExporter()
+            .Build();
+
+        try
+        {
+            Func();
+        }
+        catch (Exception)
+        {
+            // swallow the exception
+        }
+
+        Func();
```

Comment:
consider moving the unhandled (and app crashing) scenario to  only the doc, to keep this example simple and applicable to majority customers. (for eg: frameworks like asp.net core anyway handles these exceptions to avoid app crash, so its less common and best to be avoided in this doc)

Reply:
Done.